pavelib/assets.py

```diff
@@ -324,6 +324,10 @@ def update_assets(args):
         help="Skip collection of static assets",
     )
     parser.add_argument(
+        '--skip-coffee', dest='coffee', action='store_false', default=True,
```

Comment:
The attribute `.coffee` means, compile coffeescript, which is straightforward.  The switch here defaults to True, and if supplied, assigns False, which is the confusing part.